capstone/src/main/java/com/google/step/servlets/RestaurantSignUpServlet.java

```diff
@@ -0,0 +1,95 @@
+import com.google.appengine.api.datastoreDatastoreService;
+import com.google.appengine.api.datastore.DatastoreServiceFactory;
+import com.google.appengine.api.datastore.Entity;
+import com.google.appengine.api.datastore.PreparedQuery;
+import com.google.appengine.api.datastore.Query;
+import com.google.appengine.api.datastore.UserService;
+import com.google.appengine.api.datastore.UserServiceFactory;
+import com.google.appengine.api.datastore.GeoPt;
+
+import java.io.IOException;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Arrays;
+
+import UserLoginServlet.convertToJsonUsingGson;
+
+/**
+ * Servlet that will put the restaurant information in Datastore.
+ */
+@WebServlet("/restaurantSignUp")
+public class RestaurantSignUpServlet extends HttpServlet {
+
+    @Override
+    public void doPost(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        UserService userService = UserServiceFactory.getUserService();
+
+        // Only available if the user is logged in.
+        if(!userService.isUserLoggedIn()) {
+            response.setStatus(HttpServletResponse.SC_FORBIDDEN);
+            return;
+        }
+
+        String id = userService.getCurrentUser().getUserId();
+        String email = userService.getCurrentUser().getemail();
```

Comment:
Nit: "getEmail()" instead of "getemail()"

Reply:
Done